doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -476,13 +495,33 @@ Requests to the environment
       Use the :cmd:`Add @table` and :cmd:`Remove @table` commands to update the set of
       blacklisted strings.
 
-.. cmd:: Locate @qualid
+.. cmd:: Locate @locatable
+
+  .. insertprodn locatable locatable
+
+  .. prodn::
+     locatable ::= @smart_qualid
+     | Term @smart_qualid
+     | Module @qualid
+     | Ltac @qualid
+     | Library @qualid
+     | File @string
```

Comment:
See, I think It's not bad to have this here, but as far as I'm concerned this is almost an implementation detail of the grammar, and it doesn't need to be too prominent.  In particular, I wonder if it might read better as a list where Locate is repeated.

Reply:
I wouldn't unnecessarily repeat "Locate" when specifying syntax, it just makes the description more verbose and a slower read.  I expect syntax notation will be familiar to our readers.  And if some mathematician hasn't seen it, they will be able to get the hand of it quickly.